### ISSUE TITLE
Admin CMS features

### ISSUE THREAD

**Issue by lucyTKS** [SmoofCreative/manor-house#34]
Location: CMS

User: Admin

Issue: As the admin, I need to see all relevant details and be able to do X Y Z from within the CMS order section.

Notes: This is an epic and needs input from @Mario.
Similar or same as woo-comms? 

Requirements:
- List of orders
- Waiting for payment / currency or that it's been sent or received. 
- Buy & sell are different. 
- Something for exchange rates
- Something for content and articles. 
(3 sections - General pages, what's refreshing the rates + managing the orders) 
- Each currency has a buy & sell at different rates. 
- Do we bootstrap off woo-comms or build from scratch - need to weigh up benefits & cons.

**Comment by tomstarley** [comment]
@LeftoverCurrency assigning this to you for the moment, if you can think of anything else that needs to go here let me know. :)

**Comment by LeftoverCurrency** [comment]
@tomstarley this is almost complete. Only thing I can think of extra is the GDPR / privacy / cookies bit - this can be very similar to how we set up LOC last year. 

For the rest this is all exactly as we would like it. @lucyTKS I like the 3 sections bit. Keep it clean and structured, good stuff. Back to you!

**Comment by LeftoverCurrency** [comment]
@tomstarley we should have the following stages of an order:

- awaiting fulfillment   (order newly created; waiting on customer to pay or send us the currency)
- awaiting due diligence   (payment or currency received, waiting on customer to send ID / information)
- awaiting dispatch     (customer did everything required, now we need to send currency / pay)
- completed   (currency dispatched / payment sent)
- on hold   (in case of issues)
- cancelled   (if not fulfilled within timeframe or if customer wants to cancel)

**Comment by tomstarley** [comment]
@LeftoverCurrency do you need a cancelled email or is that one for the future?
